### ISSUE TITLE
Unable to reference port in feature group from Behavior Annex specification.

### ISSUE THREAD

**Issue by smithdtyler** [osate/osate2#2146]
## Summary

In 2.6.0 I was able to reference ports in feature groups as part of transition guards in the AADL behavior annex. That ability seems to have been lost in 2.6.1. @Etienne13 indicated this is likely a regression. 

See https://groups.google.com/forum/?utm_medium=email&utm_source=footer#!topic/osate/zeZozfceOqI

## Steps to Reproduce
See AADL model below.

```aadl


feature group RodentsRequired

features 

Hamster: in event data port;

    end RodentsRequired;

    

    feature group RodentsProvided

features 

Gerbil: out event data port;

    end RodentsProvided;

  

 thread group threadA

    features

      inputA: in feature group RodentsRequired;

      outputA: out feature group RodentsProvided;

  end threadA;

  

  thread group implementation threadA.impl

    subcomponents

      t0 : thread;

    annex behavior_specification {**

    states

    s0: initial state;

    s1: final state;

    transitions

    s0 -[inputA.Hamster=1]-> s1 {outputA.Gerbil:=1};

    **};

  end threadA.impl;


```
_Error message: Wrong type in value variable stop rule; expected types are: data subcomponent or data access or behavior variable or data access feature prototype or in parameter or in port or in port prototype or iterative variable_

## Environment

* **OSATE Version**: 2.6.1
* **Operating System**: Windows 10

**Comment by lwrage** [comment]
@Etienne13 Do you think you can fix this for the next release (end of Jan 2020)?

**Comment by Etienne13** [comment]
This issue should be fixed after integrating PR #2152.
